docs/services/navigation/_index.md

```diff
@@ -554,3 +554,137 @@ obstacles = await my_nav.get_obstacles()
 
 {{% /tab %}}
 {{< /tabs >}}
+
+## Concepts
+
+The following concepts are important to understand when utilizing the navigation service.
+Each concept is a type of relative or absolute measurement, taken by a [movement sensor](/components/movement-sensor/), which can then be utilized by your robot to navigate across space.
+
+Here's how to make use of the following types of measurements:
+
+- [Compass Heading](/services/navigation/#compass-heading)
+- [Orientation](/services/navigation/#orientation)
+- [Angular Velocity](/services/navigation/#angular-velocity)
+- [Position](/services/navigation/#position)
+- [Linear Acceleration](/services/navigation/#linear-acceleration)
+- [Linear Velocity](/services/navigation/#linear-velocity)
+
+### Compass Heading
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/) take compass heading measurements:
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+
+An example of a `Compass Heading` reading:
+
+``` go
+// heading is a float64 
+heading, err := gps.CompassHeading(context.Background, nil)
+```
+
+If you want to read compass headings, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+The movement sensor API's [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading) takes compass heading readings.
+Use compass heading readings to determine the *bearing* of your robot, or, the [cardinal direction](https://en.wikipedia.org/wiki/Cardinal_direction) that your robot is facing.
```

Comment:
```suggestion
Use compass heading readings to determine the *bearing* of your robot, or, the [cardinal direction](https://en.wikipedia.org/wiki/Cardinal_direction) that your robot is facing.

To read compass headings, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
Then use the movement sensor API's [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading) method to get readings from the sensor.
```
I think this order makes way more sense because it's the what and why before the how, instead of explaining "how" before the reader knows what and why.

Reply:
Do you want me to apply this change everywhere if implemented or just compass heading @JessamyT ? What do you think about usage at top @randhid I'm good w/ either way

Reply:
Ok, thank you! I'm good with that, thanks for suggesting inline everywhere so it's an easy change to make